File: Program12.java
import java.util.*;

public class Program12
{


	public static boolean DivisibleByThreeAndFive(int iNo)
	{
		int iAns1 = 0;
		int iAns2 = 0;

		iAns1 = iNo % 5;
		iAns2 = iNo % 3;


		if((iAns1 == 0) && (iAns2 == 0))
		{
			return true;
		}
		else
		{
			return false;
		}

	}

	public static void main(String[] args) {
		
		Scanner sobj = new Scanner(System.in);
		
		System.out.println("Enter the Number");
		
		int iValue = sobj.nextInt();
		
		boolean iRet = DivisibleByThreeAndFive(iValue);

		if(iRet == false)
		{
			System.out.println(iValue + " is not divisible by 5 & 3");
		}
		else
		{
			System.out.println(iValue + " is divisible by 5 & 3");
		}


		
		
				

	}

}

File: Program12.py

def DivisibleByThreeAndFive(iNo):

	iAns1 = 0

	iAns2 = 0

	iAns1 = iNo % 3
	iAns2 = iNo % 5
	if((iAns1 == 0) & (iAns2 == 0) ):
		return True
	else:
		return False
	


def main ():

	iValue = int(input("Enter Number \n"))
	
	iRet = DivisibleByThreeAndFive(iValue)

	if(iRet == False):
		print( iValue,"is not divisible by 5 & 3")
	else:
		print( iValue,"is divisible by 5 & 3")	



if __name__ == "__main__":
	main()